index.html

```diff
@@ -123,17 +131,37 @@ <h3>Step 4.</h3>
         &lt;link rel="stylesheet" href="[your-name].css"&gt;
       </code>
 
-      <h3>Step 5.</h3>
+      <h3>Step 7.</h3>
       <p>The rest is up to you, create until your completely out of ideas!</p>
 
-      <h3>Step 6.</h3>
-      <p>Once you've finished, it's time to publish it online.</p>
+      <h3>Step 8.</h3>
+      <p>Once you've finished, it's time to create a PR to your own fork.</p>
 
-      <h3>Step 7.</h3>
-      <b>Create a PR</b>
+      <h3>Step 9.</h3>
+      <p>Merge when you are happy with it, feel free to get a friend, mentor or
+        someone on your table to review it.</p>
 
-      <h3>Step 8.</h3>
-      <b>Merge when approved</b>
+      <h3>Step 10.</h3>
+      <p>Once you have merged your PR to your fork its time to get this live on
+        JuniorDev! To do this go to the <span class="mono">Pull Request</span> section in Github.
```

Comment:
Do they go to their own Fork at this bit, or to the real HacktoberFEST repo? Just confirming

Reply:
Ooo true its their own! I think you can do it from the real one too but for the sake of making it less confusing, Ill update the screenshot and copy stating own fork tyty